src/GroupRow.tsx

```diff
@@ -16,7 +16,8 @@ const groupRow = css`
       background-color: var(--rdg-header-background-color);
     }
 
-    > .${cell}:not(:last-child):not(.${cellFrozenLast}) {
+    > .${cell}:not(:last-child, .${cellFrozen}),
+    > .${cellFrozen}:has(+ .${cellFrozen}) {
```

Comment:
Does this work?
```suggestion
    > :nth-last-child(n + 1 of .${cellFrozen}) {
```

Reply:
Does not work

Reply:
`n + 2` then? 🤔

Reply:
This works. How does it work :) ?

Reply:
For all the `nth` selectors, you can use `n` with some arithmetic to select specific elements.
https://developer.mozilla.org/en-US/docs/Web/CSS/:nth-child
I don't know why `+ 2` is needed to skip the first match though.